packages/l10n/package.json

```diff
@@ -45,7 +45,7 @@
     "react": "16.10.2"
   },
   "devDependencies": {
-    "@testing-library/react-hooks": "2.0.3",
+    "@testing-library/react-hooks": "3.1.0",
```

Comment:
The standard thing about the clean up with latest rtl version. We shouldn't be affected.